tests/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtensionTest.php

```diff
@@ -133,6 +133,7 @@ class ApiPlatformExtensionTest extends TestCase
         'http_cache' => ['invalidation' => [
             'enabled' => true,
             'varnish_urls' => ['test'],
+            'xkey' => false,
```

Comment:
Is it possible to use `enabled => false` and `xkey => true`?

Reply:
if enable => false : both xkey and http_tags are disabled.
if enable => true : we can enable xkey and/or http_tags but not disable both